releasenotes/notes/remove-clone-context-3cc6f3d73f15345e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    :py:meth:`ddrace.context.Context.clone` was removed.
```

Comment:
do we have an alternative step people should take?

or can we expand this further... why is it no longer necessary?

Reply:
I updated the PR description. Users shouldn't need to clone the an existing context so there is no work around

Reply:
I feel like we still need something here.

If someone in their code today is doing `ctx.clone()` and then they see this deprecation note, what are they suppose to do?

Is it "no longer necessary" or "instead of cloning a context do ...." ?

Reply:
@Kyle-Verhoog Do you have any thoughts here. We'd like to add more context to the release note :)

Reply:
> This removes the need to clone the context which means that ddtrace will now work out of the box with async frameworks that use contextvars.